pilot/pkg/features/pilot.go

```diff
@@ -689,6 +689,12 @@ var (
 		false,
 		"If set, it allows creating inbound listeners for service ports and sidecar ingress listeners ",
 	).Get()
+
+	EnableNativeStats = env.Register(
```

Comment:
why not make this to master?

Reply:
We'll default enable in the next release so that we can deprecate Wasm version sooner. We have months to fully test.